README.md

```diff
@@ -1,8 +1,34 @@
-# React + Vite
+# Northcoders News Front-end Site
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+This project aims to replicate the behaviour of a news website, making use of the back-end database created alongside this to display information on-screen.
 
-Currently, two official plugins are available:
+## Cloning
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+To clone the project, in your terminal set to your chosen directory, run the command `git clone https://github.com/AlexMan123456/fe-nc-news`, then do `cd fe-nc-news` to set the directory of the terminal to be in the project folder.
+
+## Required dependencies
+
+To successfully run this project, you will need to install the following dependencies:
+- react (required for managing the site) (minimum version required 18.3.1)
+    - To install this, run `npm install react` in the terminal.
+- react-router-dom (required for managing the routes) (minimum version required 6.27.0)
+    - To install this, run `npm install react-router-dom` in the terminal.
+- axios (required for making requests to the back-end server) (minimum version required: 1.7.7)
+    - To install this, run `npm install axios` in the terminal.
+-  Material UI (required for the radio buttons on all articles page) (minimum version required: 6.1.6)
+    - To install this, run `npm install @mui/material @emotion/react @emotion/styled` in the terminal.
```

Comment:
The dependencies above, can just be installed by using one commend - `npm install` . That will install all the dependencies and dev dependencies in the package.json